### PR TITLE
Add new block method

### PR DESCRIPTION
**Pull request by daniel94104**
Add new file, block.py, where Block class stored.

### CODE REVIEW

File: src/block.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import datetime
+
+class Block(object):
+    def __init__(self):
+        self.index = None
+        self.time_stamp = st = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
+        self.transactions = 0
+        self.proof = None
+        self.previous_proof= None
+
```
Comment (tonylixu) [inline src/block.py]: No newline at the end of the file. A good coding practice.

File: src/block.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import datetime
+
+class Block(object):
+    def __init__(self):
+        self.index = None
+        self.time_stamp = st = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
```
Comment (tonylixu) [inline src/block.py]: - `st` is not necessary
- `ts` is not defined. You will run into error `NameError: name 'ts' is not defined`

File: src/blockchain.py [hunk 3]
```diff
@@ -35,7 +37,16 @@ def new_block(self, proof, previous_hash):
         :param previous_hash: Hash of previous Block
         :return: New Block
         """
-        pass
+        """
+        
+        """
+        index = len(self.chain)
```
Comment (tonylixu) [inline src/blockchain.py]: index should be the current `len(self.chain) + 1`, also you need to pass the index to the new block.

File: src/blockchain.py [hunk 4]
```diff
@@ -35,7 +37,16 @@ def new_block(self, proof, previous_hash):
...
+        block = Block()
+        if valid_proof(previous_hash, proof):
+            block.index = index
+            block.proof = proof
+            block.previous_proof = previous_hash
+            return block
```
Comment (tonylixu) [inline src/blockchain.py]: Remember to append the new block into the `self.chain`, and reset `current_transactions`.

**Comment by daniel94104** [comment]
Thanks for the review. Working on it.

**Pull request closed without merging**